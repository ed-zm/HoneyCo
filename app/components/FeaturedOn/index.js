import React from 'react'
import { Row, Container } from 'reactstrap'
import styled from 'styled-components'
import FeaturedAdamCarolla from './static/adam-carolla-show.png'
import FeaturedCrains from './static/crains.png'
import FeaturedNashville from './static/nashville-business-journal.svg'
import FeaturedStanding from './static/standing-with-hope.jpg'
import FeaturedTechCo from './static/tech-co.png'

const FeaturedCards = styled.div`
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: space-around;
  padding-bottom: 40px;
  margin: 0 -15px
`

const FeaturedCardWrapper = styled.div`
  margin: 20px 10px;
  img {
    -webkit-filter: grayscale(100%);
    filter: grayscale(100%);
    opacity: 0.7;
    margin: 0 15px;
  }
`
const FeaturedOn = () =>
  <Container>
    <Row>
      <FeaturedCards>
        <FeaturedCardWrapper>
          <img src={FeaturedAdamCarolla} width="180" alt="The Adam Carolla Show" />
        </FeaturedCardWrapper>
        <FeaturedCardWrapper>
          <img src={FeaturedCrains} width="180" alt="Crain's" />
        </FeaturedCardWrapper>
        <FeaturedCardWrapper>
          <img src={FeaturedNashville} width="180" alt="Nashville Business Journal" />
        </FeaturedCardWrapper>
        <FeaturedCardWrapper>
          <img src={FeaturedStanding} width="180" alt="Standing with Hope" />
        </FeaturedCardWrapper>
        <FeaturedCardWrapper>
          <img src={FeaturedTechCo} width="180" alt="Tech Co" />
        </FeaturedCardWrapper>
      </FeaturedCards>
    </Row>
  </Container>

export default FeaturedOn
