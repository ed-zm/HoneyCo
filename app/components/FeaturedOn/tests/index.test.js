import React from 'react'
import { shallow } from 'enzyme'
import FeaturedOn from '../index'

describe('<FeaturedOn />', () => {
  it('It should render the FeaturedOn Component', () => {
    const wrapper = shallow(<FeaturedOn/>)
    expect(wrapper).toMatchSnapshot()
  })
})
