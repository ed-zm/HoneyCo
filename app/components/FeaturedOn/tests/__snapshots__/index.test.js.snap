// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeaturedOn /> It should render the FeaturedOn Component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <gt
    tag="div"
>
    <bt
        tag="div"
    >
        <styled.div>
            <styled.div>
                <img
                    alt="The Adam Carolla Show"
                    src="IMAGE_MOCK"
                    width="180"
                />
            </styled.div>
            <styled.div>
                <img
                    alt="Crain's"
                    src="IMAGE_MOCK"
                    width="180"
                />
            </styled.div>
            <styled.div>
                <img
                    alt="Nashville Business Journal"
                    src="IMAGE_MOCK"
                    width="180"
                />
            </styled.div>
            <styled.div>
                <img
                    alt="Standing with Hope"
                    src="IMAGE_MOCK"
                    width="180"
                />
            </styled.div>
            <styled.div>
                <img
                    alt="Tech Co"
                    src="IMAGE_MOCK"
                    width="180"
                />
            </styled.div>
        </styled.div>
    </bt>
</gt>,
  "nodes": Array [
    <gt
      tag="div"
>
      <bt
            tag="div"
      >
            <styled.div>
                  <styled.div>
                        <img
                              alt="The Adam Carolla Show"
                              src="IMAGE_MOCK"
                              width="180"
                        />
                  </styled.div>
                  <styled.div>
                        <img
                              alt="Crain's"
                              src="IMAGE_MOCK"
                              width="180"
                        />
                  </styled.div>
                  <styled.div>
                        <img
                              alt="Nashville Business Journal"
                              src="IMAGE_MOCK"
                              width="180"
                        />
                  </styled.div>
                  <styled.div>
                        <img
                              alt="Standing with Hope"
                              src="IMAGE_MOCK"
                              width="180"
                        />
                  </styled.div>
                  <styled.div>
                        <img
                              alt="Tech Co"
                              src="IMAGE_MOCK"
                              width="180"
                        />
                  </styled.div>
            </styled.div>
      </bt>
</gt>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <FeaturedOn />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <gt
          tag="div"
>
          <bt
                    tag="div"
          >
                    <styled.div>
                              <styled.div>
                                        <img
                                                  alt="The Adam Carolla Show"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Crain's"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Nashville Business Journal"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Standing with Hope"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Tech Co"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                    </styled.div>
          </bt>
</gt>,
        "_debugID": 2,
        "_renderedOutput": <gt
          tag="div"
>
          <bt
                    tag="div"
          >
                    <styled.div>
                              <styled.div>
                                        <img
                                                  alt="The Adam Carolla Show"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Crain's"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Nashville Business Journal"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Standing with Hope"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                              <styled.div>
                                        <img
                                                  alt="Tech Co"
                                                  src="IMAGE_MOCK"
                                                  width="180"
                                        />
                              </styled.div>
                    </styled.div>
          </bt>
</gt>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <FeaturedOn />,
}
`;
